### DISCUSSION TITLE
Plan for AWS Endpoint

### DISCUSSION THREAD

**Discussion by shauryauppal** [ebhy/budgetml#5] [General]
Currently your setup seems only to promo gcp instances. Do you plan to share about deployment with budget ML on AWS instances?

**Comment by htahir1** [comment]
I've heard a few comments over the internet about it, but still unsure whether that's a route to go down on. It should be relatively trivial though.

I'd be curious to hear why would you use AWS over GCP? Is there a price difference or some other reason that I'm not aware of

**Reply by shauryauppal** [reply]
It is not just developers choice to choose a platform. It also depends on company where dev work at. Some companies prefer to keep there all platform/cloud only on AWS.

**Reply by shauryauppal** [reply]
For a Dev asking for a GCP specifically for deployment would require him to go through permission issue. 

I would rather select cortex or bentoml to avoid this pain of permissions/special request for GCP. 

I really want to try your repo. Looks really promising though, I have starred it today. Will try on local soon. 🤩

**Reply by htahir1** [reply]
Ah yes makes sense! AWS is definitely high up on the agenda :-)

**Comment by shafaypro** [comment]
Do you need any contributor regarding the AWS based setup?

**Reply by htahir1** [reply]
Sure, im more than happy to have support. I can add contributing guidelines soon and feel free to create a PR. Full disclosure, just in case: There is no monetary benefit from this :-)
